abohlib 1.0.1

abitofhelp opened the pull request:
Created via `alr publish` with `alr 2.1.0`

mosteo commented on index/ab/abohlib/abohlib-1.0.1.toml:
```diff
@@ -0,0 +1,89 @@
+[[depends-on]]
+sha2 = "^2.0.0"
+
+[[actions]]
+type = "test"
+command = [
+    "alr",
+    "exec",
+    "--",
+    "gprbuild",
+    "-P",
+    "tests.gpr",
+    "-j0",
+]
+directory = "."
+
+[[actions]]
+type = "test"
+command = [
+    "./bin/test_all",
+    "--all",
+]
```
As I mentioned above, if you move the testing to its own nested crate, depending on your main library, you can simplify this part to just a call to `alr run`.

mosteo commented on index/ab/abohlib/abohlib-1.0.1.toml:
```diff
@@ -0,0 +1,89 @@
+[configuration]
+auto_gpr_with = true
+
+[build-profiles]
+"*" = "development"
```
As clients of your library might prefer other build profiles, you should leave this out of the index manifest (unless there's an actual reason behind it).

mosteo reviewed (changes requested):
Thank you for your submission, Michael.

It seems some library is missing for linking in some platforms: https://github.com/alire-project/alire-index/actions/runs/16857914546/job/47826098839?pr=1570#step:7:829

The other comments are strong hints for improvements rather than actual errors, but those would make your crate more appealing to potential users.

abitofhelp commented:
I have added platform-specific code for Linux, macOS, and Windows. I have removed the trendy_test package, which was not being used. I am closing this pull request and will generate a new one for v1.1.0.
